package.json

```diff
@@ -79,7 +79,8 @@
 				{
 					"varsIgnorePattern": "^React$"
 				}
-			]
+			],
+			"@typescript-eslint/no-non-null-assertion": "off"
```

Comment:
Just upgrade to the latest version of https://github.com/xojs/eslint-config-xo-typescript and you don't have to turn this rule off.

Reply:
done